src/Traits/PersonTrait.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+/*
+ * Copyright (c) 2021 Heimrich & Hannot GmbH
+ *
+ * @license LGPL-3.0-or-later
+ */
+
+namespace HeimrichHannot\UtilsBundle\Traits;
+
+use Contao\Model\Collection;
+use Contao\StringUtil;
+use HeimrichHannot\UtilsBundle\Model\ModelUtil;
+
+/**
+ * Trait PersonTrait.
+ *
+ * @param ModelUtil $modelUtil
+ */
+trait PersonTrait
+{
+    /**
+     * @return array
+     *               returns empty array or array of group Models
+     */
+    public function getActiveGroups(int $userId): ?Collection
+    {
+        if (!$userModel = $this->modelUtil->findModelInstanceByPk($this::TABLE, $userId)) {
```

Comment:
Bitte explizit auf null prüfen (immer!). Hier ist es sogar problematisch, da evtl. das Ausrufezeichen höhere Prio hat. Bitte immer folgendermaßen prüfen:

`if (null = ($model = $this->modelUtil->...))) {`

Reply:
Null-Prüfung ist hier nicht notwendig, da Collection in dem Fall immer true, null immer false ist.